Sources/StreamChatUI/ChatChannelList/ChatChannelListCollectionViewSkeletonCell.swift

```diff
@@ -0,0 +1,65 @@
+//
```

Comment:
All headers in the PR should be updated to the standard Stream header.

Reply:
Weird, this is usually automatic when committing 🤔

Reply:
Was having some issues with python that Nuno helped me to address and was able to fix to add the Stream header on the new files.